ocaml/xapi/certificates.ml

```diff
@@ -146,7 +146,7 @@ let host_uninstall is_cert ~name =
   debug "Uninstalling %s %s" (get_type is_cert) name;
   try
     Sys.remove filename;
-    rehash()
+    ignore(execute_command_get_output "/opt/xensource/bin/update-ca-bundle.sh" [])
```

Comment:
If rehash is not used anymore it should be deleted.

Reply:
rehash is also used here: https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/certificates.ml#L110

Reply:
But that code would have no effect anymore since we use the certificate bundle instead of directory, should the code be removed?